packages/react-icons/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@coveord/plasma-react-icons",
-    "version": "45.4.0",
+    "version": "46.0.2-next.0",
```

Comment:
not 47?

Reply:
Jenkins takes care of bumping the version, not I